docs/index.md

```diff
@@ -1,4 +1,86 @@
-# Welcome to the PebbleOS documentation!
+```{raw} html
+<div style="display: none;">
+```
+# PebbleOS documentation
+```{raw} html
+</div>
+```
+
+::::{grid} 2
+:reverse:
+
+:::{grid-item}
+:columns: 4
+:class: sd-m-auto
+
+<img src="_static/images/logo.svg" />
+
+:::
+
+:::{grid-item}
+:columns: 8
+
+```{div} sd-fs-3 home-title
+Welcome to the PebbleOS documentation site!
+```
+
+PebbleOS is a lightweight, power-efficient operating system originally developed for Pebble smartwatches and now [released open source](https://opensource.googleblog.com/2025/01/see-code-that-powered-pebble-smartwatches.html).
+It supports custom watchfaces and apps using C and JavaScript.
+Optimized for e-paper displays and long battery life, it features Bluetooth sync, a timeline interface, and a strong developer ecosystem through an open SDK.
```

Comment:
MIP (memory in pixel), not e-paper, they're different technologies and we probably shouldn't perpetuate the incorrect naming.

Reply:
e-paper is Eric's name for anything transflective :-) e-ink is the specific transflective electrophoretic display technology that e-paper got confused with

Reply:
Ah, I did misread this as e-ink actually. I think Sharp possibly calls it e-paper in some places, but either way, we should probably just use the actual name for the display technology.